src/scss/toolbar.scss

```diff
@@ -228,7 +228,6 @@
       display: block;
       text-align: center;
       font-size: $xsmall;
-      margin-top: -2px;
```

Comment:
removed because this looked off with the fullscreen icon which seems to use more vertical space than the filter icon